packages/zoltan2/src/problems/Zoltan2_ColoringProblem.hpp

```diff
@@ -135,9 +135,17 @@ class ColoringProblem : public Problem<Adapter>
   {
     RCP<Teuchos::StringValidator> color_method_Validator = Teuchos::rcp(
       new Teuchos::StringValidator(
-        Teuchos::tuple<std::string>( "SerialGreedy" )));
+        Teuchos::tuple<std::string>( "SerialGreedy","Hybrid","2GL" )));
     pl.set("color_method", "SerialGreedy", "coloring algorithm",
      color_method_Validator);
+    pl.set("Hybrid_batch_size",-1,"Batch size for distributed coloring, default is all verts",
```

Comment:
Is batch size here the same as superstep size in the old Zoltan coloring? Please add a comment in the code to clarify.

Reply:
I will clean up those arguments, as they are no longer used.